src/renderer/components/test-utils/get-application-builder.tsx

```diff
@@ -181,7 +185,7 @@ export const getApplicationBuilder = () => {
     setMenuItems: (items) => {
       trayMenuItemsStateFake = items;
     },
-    setIconPath: (path) => {
+    setIconPath: (path: string) => {
```

Comment:
nit: adding this type here is not necessary because it is declared above on line 73